core/src/test/java/org/elasticsearch/index/query/FuzzyQueryBuilderTests.java

```diff
@@ -38,6 +39,14 @@
 
 public class FuzzyQueryBuilderTests extends AbstractQueryTestCase<FuzzyQueryBuilder> {
 
+    /**
+     * all tests create deprecation warnings through the builders constructor, those are
+     * checked once here after test it run
```

Comment:
what do you mean by "the builders constructor" ? Should it be "after each test is run" ?